sdk/containerregistry/ci.yml

```diff
@@ -0,0 +1,51 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - master
+      - main
+      - hotfix/*
+      - release/*
+  paths:
```

Comment:
Let's use the pattern outlined in AppConfiguration, KeyVault, and Storage where we only use `include` paths as those are easier to maintain over time.

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/ci.yml